pkg/account/handler.go

```diff
@@ -97,24 +89,22 @@ func (h *AccountHandler) LoginAccount(c echo.Context) error {
 // @Description host will send a verification code to email, need response with verification code
 // @Tags Account
 // @Produce json
-// @Param email formData string true "user e-mail"
-// @param vericode formData string true "verification code sent by user"
-// @Param paswd formData string true "user password"
-// @Success 200 {string} string "{"msg": "Successfully modified"}"
-// @Failure 400 {string} string "{"error": "Invali E-mail Address"}"
-// @Router /case/PastHistory/{id} [PATCH]
+// @Param Email path string true "user e-mail"
+// @param VeriCode path string true "verification code sent by user"
+// @Param Passwd path string true "user password"
+// @Success 200 {string} api.ReturnedData{data=string}
+// @Failure 400 {string} api.ReturnedData{data=string}
+// @Router /account/{id} [POST]
```

Comment:
The router conflicts with `ModifyPasswd`. What about `/account/{id}/reset [POST]`?